packages/features/bookings/lib/get-booking.ts

```diff
@@ -224,3 +224,15 @@ export const getBookingForSeatedEvent = async (uid: string) => {
   };
   return result;
 };
+
+export const getMultipleDurationValue = (
+  multipleDurationConfig: number[] | undefined,
+  queryDuration: string | string[] | undefined,
+  defaultValue: number
+) => {
+  return multipleDurationConfig
+    ? multipleDurationConfig.includes(Number(queryDuration))
+      ? Number(queryDuration)
+      : defaultValue
+    : null;
+};
```

Comment:
Legibility nitpick

[Prefer Early Returns](https://handbook.cal.com/engineering/best-practices/prefer-early-returns) 

```suggestion
export const getMultipleDurationValue = (
  multipleDurationConfig: number[] | undefined,
  queryDuration: string | string[] | undefined,
  defaultValue: number
) => {
  if (!multipleDurationConfig) return null;
  if (multipleDurationConfig.includes(Number(queryDuration))) return Number(queryDuration);
  return defaultValue;
};
```